src/components/v2/Pagination/usePagination.ts

```diff
@@ -54,7 +54,7 @@ export function usePagination({
     : activePageIndex + halfOfPagesCount;
 
   const handlePageChange = (pageIndex: number) => {
-    onChange(pageIndex);
+    onChange(pageIndex + 1);
```

Comment:
I don't quite understand the +1 here. I can see pagination works perfectly fine on the History page so I'm sure this is needed, but I'm curious as to why

Reply:
Ahh right. I think this logic of defining the page number displayed to the user should be kept inside the `History` page component then, so that we always use a zero based indexation in our logic (especially since the `onChange` callback of `usePagination` names its param `newPageIndex`).